### ISSUE TITLE
Legend goes invisible when attempting to display large amounts of datasets

### ISSUE THREAD

**Issue by Danielv123** [Tsur/canvasjs#8]
I have a graph with 50 datasets and 5k datapoints each. I want to be able to toggle the display of some of the sets with the legend as shown in http://canvasjs.com/docs/charts/how-to/hide-unhide-data-series-chart-legend-click/

It works except for the fact that large amounts of datasets makes the legend buttons invisible. Only the first legend item is visible (on the left side) while the ones towards the right is clickable, overlapping and invisible.

**Comment by cedriccouret** [comment]
Hi, I'm facing the same issue. Any news on that point ?

Thanks

**Comment by Danielv123** [comment]
No, I haven't found a solution. The best I can recommend is writing your own dropdown menu (or similar) for toggling things on the legend.
